### PR TITLE
Add SBT devrun command to readme

### PR DESCRIPTION
**Pull request by desbo**
A minor fix, but actually quite an important step in running locally.

### CODE REVIEW

**Comment by alexduf** [comment]
I think it's a good idea, but I would add that a bit lower in the "Run" section. Something saying if you want to run it manually blahblah ?

**Comment by desbo** [comment]
Good call, I've reshuffled things a bit in the readme

File: README.md [hunk 1]
```diff
@@ -99,6 +88,23 @@ This will start the Play application, which usually listens on port `9111`. Maki
 
 To make the site reachable as `contributions.thegulocal.com` (necessary for register/sign-in functionality) you then need to make sure NGINX is configured and running as described in [`/nginx/README.md`](./nginx/README.md).
 
+
+### Running the play app manuall
+```
+sbt devrun
```
Comment (AWare) [inline README.md:94]: ./start-frontend.sh?

File: README.md [hunk 2]
```diff
@@ -99,6 +88,23 @@ This will start the Play application, which usually listens on port `9111`. Maki
...
+```
+
+### Watch files for changes
+
+```
+npm run watch
+```
+
+This runs [webpack-dev-server](https://webpack.github.io/docs/webpack-dev-server.html) on port 7777, proxying requests to 9111. In this case, the bundled JS is stored in memory, which allows for much faster recompilation on changes, but the main.js file won't be created. You can also access localhost:7777/webpack-dev-server/\<route\> for a live reload environment.
```
Comment (AWare) [inline README.md:103]: This approach doesn't have ssl, which will be more important when we do more on the thank you page as that breaks play sessions. There's a little work remaining to properly proxy this securely, but if you add giraffe.thegulocal.com to HOSTS it mostly works.

**Review by AWare — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alexduf) [reply README.md:103]: yeah I think there's a broader conversation to have on how to start the dev server now we have webpack.
Could that be integrated with sbt? or should we have a local architecture with Nginx -> webpack -> play ?

File: README.md (hunk 1 above)
Comment (desbo) [reply README.md:94]: look 10 lines up

File: README.md (hunk 2 above)
Comment (desbo) [reply README.md:103]: Yeah, it seems a little confusing that there's nginx hosts for `contribute.thegulocal.com` and `giraffe.thegulocal.com`. We should rework how we run things locally for sure, and also tidy up the nginx config and decide on one hostname to use for development – that'll give us a much cleaner environment to document.

File: README.md (hunk 1 above)
Comment (AWare) [reply README.md:94]: My bad. Sorry.

**Comment by desbo** [comment]
I'm going to close this because I think there's a fair bit more work to do with the dev workflow, so we might as well re-do the docs at the same time as that.

**Pull request closed without merging**